packages/esm-ward-app/src/config-schema.ts

```diff
@@ -1,5 +1,120 @@
-import { type ConfigSchema } from '@openmrs/esm-framework';
+import { type PersonAddress, Type, validator, validators, type ConfigSchema } from '@openmrs/esm-framework';
+import { type BentoElementType, bentoElementTypes } from './types';
 
-export const configSchema: ConfigSchema = {};
+const defaultWardPatientCard: WardPatientCardDefinition = {
+  card: {
+    id: 'default-card',
+    header: ['bed-number', 'patient-name', 'patient-age', 'patient-address'],
+  },
+  appliedTo: null,
+};
 
-export interface ConfigObject {}
+export const defaultBentoElementConfig: BentoElementConfig = {
+  addressFields: ['cityVillage', 'country'],
+};
+
+export const builtInBentoElements: BentoElementType[] = [
+  'bed-number',
+  'patient-name',
+  'patient-age',
+  'patient-address',
+  'admission-time',
+];
+
+export const configSchema: ConfigSchema = {
+  wardPatientCards: {
+    _description: 'Configure the display of ward patient cards',
+    bentoElementDefinitions: {
```

Comment:
What is a `bentoElementDefinition`? I'm not entirely sure this nomenclature is clear.

Reply:
I think something like `patientCardElement` might be clearer.

Reply:
Yeah, it's the individual elements that can show up within a patient card row (like bed number, patient name, patiant age, patient address). I'll change it to `PatientCardRowElement`.

Reply:
I see where you're coming from with `PatientCardRowElement` @chibongho but unfortunately I think the formal logic of the name does not align with intuition—I think it is very easy to read `PatientCardRowElement` and think the element is supposed to be a row. I think @mseaton 's suggestion of `patientCardElement` is clearest.

Reply:
So this would be `patientCardElementDefinitions`